### PR TITLE
Fix travis config

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

**Comment by dhensby** [comment]
Errors

**Comment by tractorcow** [comment]
Ok, trying again.

**Comment by dhensby** [comment]
Still no build

**Comment by tractorcow** [comment]
Oh, I need to update phpunit.xml.dist

**Comment by tractorcow** [comment]
Pushed up new `phpunit.xml.dist`

File: .travis.yml [hunk 1]
```diff
@@ -6,15 +6,26 @@ php:
...
+    - CORE_RELEASE="master"
 
 before_script:
- - composer self-update || true
- - phpenv rehash
- - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
- - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
- - cd ~/builds/ss
+# Init PHP
+  - export CORE_RELEASE=$TRAVIS_BRANCH
```
Comment (dhensby) [inline .travis.yml]: Why do we define this ENV var twice?

**Comment by tractorcow** [comment]
Doesn't support transactions, I guess.

**Comment by tractorcow** [comment]
Removed the dupe ENV. Can be merged now?

**Comment by tractorcow** [comment]
Yay thanks @chillu

**Pull request merged**